content/yaml/building-a-native-ios-app.md

```diff
@@ -22,12 +22,28 @@ For building an unsigned iOS app (.app), you need to run the following command i
                        CODE_SIGNING_REQUIRED=NO \
                        CODE_SIGNING_ALLOWED=NO
 
+If you don't have a workspace, use `-project "MyXcodeProject.xcodeproj"` instead of the `-workspace "MyXcodeWorkspace.xcworkspace"` option.
+
+Your artifact will be generated at the default Xcode path (unless you explicitly specify another path using the `CONFIGURATION_BUILD_DIR` option) by adding the following pattern in the `artifacts` section of `codemagic.yaml`:
+
+    $HOME/Library/Developer/Xcode/DerivedData/**/Build/**/*.app
+
+If you have Xcode Debugging Symbols enabled, dSYM file will be generated in the same directory as the app and can be accessed with pattern:
+
+    $HOME/Library/Developer/Xcode/DerivedData/**/Build/**/*.dSYM
+
 ## Building a native iOS app archive (.ipa)
 
 For building an archived iOS app (.ipa), you need to run the following command in the scripts section:
 
     - xcode-project build-ipa --project "$FCI_BUILD_DIR/MyXcodeProject.xcodeproj" --scheme "MyScheme"
```

Comment:
This example only covers the possibility to use Xcode projecet, but actually one can build ipas from workspaces too. See [--project](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/xcode-project/build-ipa.md#--projectxcode_project_path) and [--workspace](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/xcode-project/build-ipa.md#--workspacexcode_workspace_path) arguments.

Reply:
added here 79cf5b8